pagopa/pn-frontend
# Feat/pn 2869 - delegate infos under viewed state

## Short description
Added delegate information under VIEWED state in notification timeline

## List of changes proposed in this pull request
- In the mapper that adjusts the BE data, I set for the VIEWED macro-step a new property (recipient property) that contains the "denomination (tax id)" string
- In the utility method, that returns for each timeline step the title and the description, I added a new parameter (recipient parameter) that contains the  "denomination (tax id)" string
- In the localization files, I updated the state strings to accept dynamic data

## How to test
Create a notification and delegate to an another user. Login with that user and view the notificaion. Return to the previous user and check that in notification timeline there is the new description under VIEWED step
